bootstrap

```diff
@@ -63,4 +63,12 @@ echo "Generating file lists: ${VARS_FILE}"
   printf "TSS2_MU_SRC = \$(TSS2_MU_C) \$(TSS2_MU_H)"
 ) > ${VARS_FILE}
 
+# Generate fuzz tests unless environment variable NO_GEN_FUZZ is set to 1
```

Comment:
I think, the default should be "fuzz-free", i.e. for maintainers and users.
This I'd like to see the default case reversed.

Reply:
Will do